libbpf-tools/offcputime.c

```diff
@@ -314,6 +314,12 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
+	if (probe_tp_btf("sched_switch")) {
+		bpf_program__set_autoload(obj->progs.sched_switch_tp, false);
+	} else {
+		bpf_program__set_autoload(obj->progs.sched_switch_btf, false);
+	}
```

Comment:
We are following kernel code style guide, so drop the surrounding `{}`.

Reply:
> We are following kernel code style guide, so drop the surrounding `{}`.

Do you mean to write it like this? 
```
	bpf_program__set_autoload(obj->progs.sched_switch_btf, false);
	if (probe_tp_btf("sched_switch")) {
		bpf_program__set_autoload(obj->progs.sched_switch_tp, false);
		bpf_program__set_autoload(obj->progs.sched_switch_btf, true);
	}
```